### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by tech2mar** [Blackymas/NSPanel_HA_Blueprint#183] [General]
Hi, got some question, how too update this?

Thanks for the good work

**Comment by Blackymas** [comment]
hi - We will be writing a documentary on the wiki about this in the next few days.

**Reply by tech2mar** [reply]
Thanks, that would be great.

**Reply by Blackymas** [reply]
here the steps

1. update ESPhome -> validate -> install
2. HA -> Device -> TFT Upload
3. copy nspanel_blueprint.yaml -> add in the right document -> restart automations
4. automations -> NSPanel -> language and weather selected again -> automation saved
5. reboot NSPanel

**Reply by tech2mar** [reply]
So i don't need to update the blueprint?

Thanks

**Reply by Blackymas** [reply]
se point 3 -yes you need to update the blueprint. All 3 Files are need a update

**Comment by tech2mar** [comment]
Thanks worked like a charm, it would be great to update it only over one click.

But that's such a nice work , respect.very well made

**Reply by Blackymas** [reply]
Thanks! :)
